hack/jenkins/e2e.sh

```diff
@@ -485,8 +485,15 @@ case ${JOB_NAME} in
     ;;
 
   # kubernetes-upgrade-gke
-  # TODO(14771): this solution won't work once we start releasing 1.1
-  # releases, because it assumes that the release tars are 1.0.
+  #
+  # This suite:
+  #
+  # 1. launches a cluster at latest,
```

Comment:
yeah, changing this to read "release/latest" (and also for step 3/5) would make it clearer what this means